reference/approver.yaml

```diff
@@ -1429,6 +1429,178 @@ components:
           description: >
             List of project ids for which this category can be used. If
             project_ids is null, then this category can be used for any project.
+    expense_policy_rule_minimal:
+      type: object
+      properties:
+        id:
+          type: string
+          example: tpr1234abcd
+          description: This represents the id of the expense policy rule.
+        description:
+          allOf:
+            - $ref: '#/components/schemas/description'
+          description: |
+            This represents the description of the expense policy rule.
+          example: Receipt is mandatory for expense above $50
+        action_show_warning:
+          type: boolean
+          description: >
+            If it is set to true, then a warning is shown to the employee,
+            whenever he attempts to create an expense that violates this expense
+            policy rule.
+          example: true
+    expense_policy_violating_expenses_query_params:
+      type: object
+      description: >
+        Query params that can be combined and used to fetch the violating
```

Comment:
Query params that can be combined and used to fetch the violating expenses. Violating expenses means/are .......

Reply:
An expense may be violating an expense policy rule due to a group of other expenses. These query params can be used to fetch those "violating" expenses because of which this expense is causing a violation.